src/index.html

```diff
@@ -17,11 +17,481 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/styles.css"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <!--#region Header-->
+    <header class="header">
+      <div class="header__logo">
+        <a
+          class="header__logo-link logo-1200"
+          href="index.html"
+        >
+          <img
+            src="images/logo.png"
+            alt="Logo"
+          />
+        </a>
+      </div>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
```

Comment:
Your navigation links all point to the same page (index.html). If this is just a placeholder, that's fine, but don't forget to replace it with actual links to different pages.